Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/EdgeBrowserContentLocator.cs

```diff
@@ -104,20 +104,22 @@ private AutomationElement ResolveByChangingWindowTitle (IWebDriver driver, IRead
 
     private Rectangle ResolveBoundsFromWindow (AutomationElement window)
     {
+      Func<AutomationElement> elementQuery = () => window.FindFirst (

```

Comment:
R# is telling me to make this a local function. Is there a particular reason why you made it this way?

Reply:
The meaning of this is to prevent weird caching effects and to make sure the element is newly evaluated every time, correct?

Reply:
That's correct.